### ISSUE TITLE
Add ore dictionary entries for the worktable tools by material

### ISSUE THREAD

**Issue by ChiriVulpes** [codetaylor/artisan-worktables#74]
Ore dictionaries for the tools, such as `toolFlint` `toolStone`, `toolDiamond` would be nice. Not useful without #69 probably, except for maybe Game Stages stuff? With #69, you could make auto-imported recipes for them that use a different table by looping through them and checking which tool type ore dictionary they're a part of.

Example, using #69:

```js
function getTier(item as IItemStack) as string {
  if (<ore:toolIron> has item) {
    return "blacksmith";
  }
  [etc]
}

val toolMap = {
  blacksmith: <ore:blacksmiths_hammer>,
  [etc]
} as IItemStack[string];

for item in <ore:carpenters_hammer>.items {
  val tier = getTier(item);
  recipes.remove(item);
  Worktable.createRecipeBuilder(tier)
      .copyRecipeLayout(item)
      .addTool(toolMap[tier], 1)
      .addOutput(item)
      .create();
}
```

**Comment by codetaylor** [comment]
Added ore dict groups for tools by material type; the ore dict keys can be changed in the tool material json config file.

**Comment by codetaylor** [comment]
Available in the alpha snapshot 1.12.2-1.15.25-10-gcce9f8f

Will be available in the beta 1.12.2-1.16.x
